ERA-11882: Subjects/Subject groups are displayed incorrectly when the last location string is too long.

luixlive opened the pull request:
### What does this PR do?
Set humanize-duration largest option to 2 so we don't have long time ago texts.

### How does it look
**Before:**
<img width="1920" height="991" alt="image" src="https://github.com/user-attachments/assets/339c7d0d-11f5-49f8-bcdb-c0bb3fb4d571" />


**After:**
<img width="1920" height="987" alt="image" src="https://github.com/user-attachments/assets/e02aace2-3032-48c9-b52e-e7f6a5008d96" />


### Relevant link(s)
* [ERA-11882](https://allenai.atlassian.net/browse/ERA-11882)

### Any background context you want to provide(if applicable)
Here is the official NPM docs for `humanize-duration`: https://www.npmjs.com/package/humanize-duration .
In that page, search `largest` option: "Integer representing the maximum number of units to use.". Pretty straightforward.

Since it's a trivial / safe change, and I saw the fix working directly in the affected by running it locally, I didn't create an environment.


[ERA-11882]: https://allenai.atlassian.net/browse/ERA-11882?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

luixlive committed:
ERA-11882: Set humanize-duration largest option to 2 so we don't have long time ago texts

luixlive committed:
Merge branch 'ERA-11901' into ERA-11882

luixlive committed:
ERA-11882: Fix test

gaboDev reviewed (approved):
LGTM

Also, it would be worth it to add flexibility to the map layer title with CSS.

Not sure of that would have side effects in other parts of the app so I'm approving since the main goal was accomplished

luixlive commented:
> Also, it would be worth it to add flexibility to the map layer title with CSS.

Happy to work on anything that improves our list. But could you explain a bit more? What do you mean with _adding flexibility to the map layer title_ @gaboDev ?

gaboDev commented:
> > Also, it would be worth it to add flexibility to the map layer title with CSS.
> 
> Happy to work on anything that improves our list. But could you explain a bit more? What do you mean with _adding flexibility to the map layer title_ @gaboDev ?

When there is no enough space the title tends to break chaotically instead showing elipsis dots or break gracefully 

@luixlive

luixlive committed:
Merge branch 'develop' into ERA-11882

luixlive committed:
ERA-11882: Add ellipsis to long item names in the subjects tab of map layers and set the name as the title

luixlive commented:
Done @gaboDev 
<img width="1920" height="987" alt="image" src="https://github.com/user-attachments/assets/14504568-486a-4cc7-8e1b-9e97a9c90abc" />

I added ellipsis and if the user hover they will see the tooltip with the complete name (I added it as the title of the wrapper div).
